docs/diagrams/StorageFileClassDiagram.puml

```diff
@@ -0,0 +1,37 @@
+@startuml
+'https://plantuml.com/class-diagram
+
+skinparam classAttributeIconSize 0
+skinparam guillemet false
+
+class StorageFile
+StorageFile : -fileName
+StorageFile : -dataFileRelativePath
+StorageFile : +saveData()
+StorageFile : +loadData()
+StorageFile : +writeDataToFile()
+StorageFile : +get*InFileFormat()
+StorageFile : +transferDataFromFileToList()
+StorageFile : +load*()
+StorageFile : +create*()
+StorageFile : <u>+setFields()
+
+
+
+class Logger
+
+class ContactsManager
+
+class ExpenseTracker
+
+class StudyManager
+
+class AllOnUs
+
+AllOnUs --> "1" StorageFile
+StorageFile --> "1" Logger
+StorageFile "1" <--> "1" StudyManager
+StorageFile "1" <--> "1" ContactsManager
+StorageFile "1" <--> "1" ExpenseTracker
```

Comment:
Is there a reason why you don't include the navigability from `AllOnUs` to the manager/tracker classes? If these are less relevant to this diagram, is it maybe better to leave out the `AllOnUs` class entirely to avoid confusion?